addon/components/frost-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Component definition for the frost-error component
+ */
+import layout from '../templates/components/frost-error'
+import Component from './frost-component'
+import {PropTypes} from 'ember-prop-types'
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  layout,
+  tagName: 'div',
+
+  // == PropTypes =============================================================
+
+  /**
+   * Properties for this component. Options are expected to be (potentially)
+   * passed in to the component. State properties are *not* expected to be
+   * passed in/overwritten.
+   */
+  propTypes: {
+    // options
+    description: PropTypes.string,
+    suggestion: PropTypes.string,
+    errorCode: PropTypes.number,
+    errorTitle: PropTypes.string,
+    errorMessage: PropTypes.string,
+    errorDetails: PropTypes.string,
```

Comment:
I think most of the above should be required, right?  Maybe we could get away with `suggestion` (others?) being optional, but it probably doesn't make sense to not require information that would always be available in the case of an HTTP response